src/client/redux/categories/operations.js

```diff
@@ -5,7 +5,7 @@ const fetchCategories = createAsyncThunk(
   'categories/fetchAll',
   async (_, thunkAPI) => {
     try {
-      const response = await axios.get('/categories');
+      const response = await axios.get('/recipe/category-list');
```

Comment:
Im not sure that we have endpoint like this. If you want to use it implement that endpoint and declare logic.

Reply:
Those endpoints are from excel file.